scipy/stats/_continuous_distns.py

```diff
@@ -3042,6 +3084,17 @@ def _stats(self):
     def _entropy(self):
         return _EULER + 1.
 
+    def fit(self, data, *args, **kwds):
+        # The fit method of `gumbel_r` can be used for this distribution with
+        # small modifications. The process to do this is
+        # 1. pass the sign negated data into `gumbel_r.fit`
+        # 2. negate the sign of the resulting location, leaving the scale
+        #    unmodified.
+        # `gumbel_r.fit` holds necessary input checks.
+
+        loc_r, scale_r, = gumbel_r.fit(-data, *args, **kwds)
```

Comment:
`data` comes directly from the user, and it might be a list, so we have to pass it through `asarray` before we can negate it with `-`.
```suggestion
        loc_r, scale_r, = gumbel_r.fit(-np.asarray(data), *args, **kwds)
```